scripts/visualise_results.py

```diff
@@ -22,6 +22,10 @@ def plot_data(df):
 
     folders = sorted(glob(f"{parsed_variables['output_dir']}/*"))
     results = get_results(folders, parsed_variables['number_of_days'])
-    data_frame = DataFrame(results, columns=["DATE", "GET", "POST", "AVERAGE"])
+    result_fields = []
+    for result in results:
+        result_fields.append([result[0], result[1].average_get, result[1].average_post, result[1].average_total])
```

Comment:
Could be a comphrension
```suggestion
    result_fields = [
        [
            result[0],
            result[1].average_get,
            result[1].average_post,
            result[1].average_total,
        ]
        for result in results
    ]
```